### ISSUE TITLE
Using a seperate list for choices?

### ISSUE THREAD

**Issue by Sentr1k** [CITGuru/PyInquirer#69]
Say I have a list that's user created, or I'm storing options for restoring save points in my program, I need keys that can be displayed.

Something like: 
`list = [Question1, 2 ,3 ,4 ,5]`
`questions = [`
`            {`
 `               'type': 'list',`
  `              'name': 'saves',`
   `             'message': "What save would you like to load?",`
    `            'choices': list`
     `        }`
      `  ]`

**Comment by Shinyhero36** [comment]
Yeah I struggle doing the same, did you find a solution? @Sentr1k

**Comment by Shinyhero36** [comment]
I finally found how to do it:
``python
here is the question/answer
questions = [
		{
		'type': 'list',
		'name': 'file',
		'message': 'Select file to open',
		'choices': list_files(projects_path),
		}
		]
answers = prompt(questions, style=custom_style_2)
``
and here is my function with **parameter**
``python
def list_files(projects_path):
	files = []
	for file in listdir(str(projects_path)):
		files.append(file)
	return files
``
